devel/apache-ivy/Portfile

```diff
@@ -26,10 +26,8 @@ worksrcdir          ${name}-${version}
 
 depends_build       port:apache-ant
 
-checksums           md5     53f27c8ef5da2eff5039efa6c45b84e7 \
-                    sha1    244213bd0d0f344e17678ccbc72b341db6c5b9a3 \
-                    sha256  20f9ba64b6f24328497394d8b3e24b8e15e12ad230958be9c76d6f8cccf081de \
```

Comment:
Please leave the MD5 and SHA1 checksums; upstream publishes them, so it’s useful for double-checking.